### DISCUSSION TITLE
Uncaught Error: Call to undefined function UniFi_API\\json_encode()

### DISCUSSION THREAD

**Discussion by zx5000** [Art-of-WiFi/UniFi-API-client#84] [General]
I keep installing php7.3-json but it won't show up in the modules list (php -m) or php -i
`pi@ep-pi:~/unify/vendor $ php -i | grep -in json`
`24:/etc/php/7.3/cli/conf.d/20-json.ini,`
Raspbian 10 (Buster)

I'm hoping there's a sage around here who can help me get this module enabled.

Thanks in advance.

Jeff

**Comment by astro82** [comment]
I am also using Raspbian Buster with PHP 7.4 and no such problem. So why not try PHP 7.4?

**Comment by zx5000** [comment]
I fixed it by re-installing raspbian buster and following the same instructions to enable the module. I didn't have time to experiment anymore. PHP 7.4 would not install due to problems with PHP 7.3. Now running successfully with PHP 7.3.19-1~deb10u1

IMHO PHP and Python are very brittle environments because it's so easy to break them.

**Reply by astro82** [reply]
Glad you found a solution. 

Raspbian Buster is not so old and I had a problem too but with PHP 7.1 (need older scripts from RPi3) running on it. So uninstalled v7.1 and switched to 7.4 without need to reinstall. Works perfectly with both client and browser...
